CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ All notable changes to this project will be documented in this file.
 * ### ALL
     * #### Added
     * #### Changed
+        * Support specifying custom type for function parameter without "struct " prefix
```

Comment:
only publicly facing changes should be in this file
In this case you can remove

Reply:
#1713

Reply:
Thanks for pointing this out, I have reverted the changes on CHANGELOG.md with the latest commit.